### ISSUE TITLE
Apps from this role are installed instead of the ones declared locally

### ISSUE THREAD

**Issue by alexbartisro** [geerlingguy/ansible-role-mas#8]
So I have a .sh script which generates my yml

```
#!/bin/bash
read -p "Mac App Store username: " name
name=${name}

read -p "Mac App Store password: " password
password=${password}

echo "mas_email: $name
mas_password: $password
mas_installed_apps:
  - { id: 497799835, name: \"Xcode (8.x.x)\" }" | tee mas.yml

```

So My final yml is:
```
mas_email: app store username
mas_password: app store password
mas_installed_apps:
  - { id: 497799835, name: \"Xcode (8.x.x)\" }
```

And I call it from:
```
---
- hosts: all
  connection: local

  vars_files:
    - mas.yml

  roles:
    - geerlingguy.homebrew
    - geerlingguy.mas
```
as
```
ansible-playbook -i inventory --ask-become-pass xcode_req.yml
```

But the issue is when it installs, it installs all 4 mas apps from this example: xcode, blackmagic speed test, etc.

Any idea why?

**Comment by geerlingguy** [comment]
Maybe the quoting in the `mas_installed_apps` is causing an issue?

Can you throw in a section above the `roles` section:

```yaml
  pre_tasks:
    - debug: var=mas_installed_apps
```

And then see what it outputs?

**Comment by alexbartisro** [comment]
I've found the problem.
After installing xcode I went and installed another playbook called main.yml. The second one also had 

```
  roles:
    - geerlingguy.homebrew
    - geerlingguy.mas
```

But did not install anything from the appstore, just casks and taps. Apparently this triggered geerlingguy.mas to install the 4 apps from your repo here: xcode, blackmagic, etc.

Removing ```    - geerlingguy.mas``` from the second .yml fixed the problem.
